CHANGELOG.md

```diff
@@ -72,18 +62,42 @@ All notable changes to this project will be documented in this file.
 
 ### Deprecated
 
--   #1820 : Deprecated unused decorator `@lambdify`
 -   #1786 : Remove support for `real` and `integer` as type annotations.
 -   #1812 : Stop allowing multiple main blocks inside a module.
 -   \[INTERNALS\] Remove property `ast.basic.TypedAstNode.precision`.
 -   \[INTERNALS\] Remove class `ast.datatypes.DataType` (replaced by `ast.datatypes.PrimitiveType` and `ast.datatypes.PyccelType`).
 -   \[INTERNALS\] Remove unused properties `prefix` and `alias` from `CustomDataType`.
 -   \[INTERNALS\] Remove `ast.basic.TypedAstNode._dtype`. The datatype can still be accessed as it is contained within the class type.
--   \[INTERNALS\] Removed unused and undocumented function `get_function_from_ast`.
 -   \[INTERNALS\] Remove unused parameters `expr`, `status` and `like` from `pyccel.ast.core.Assign`.
 -   \[INTERNALS\] Remove `pyccel.ast.utilities.builtin_functions`.
 -   \[INTERNALS\] Remove unused/unnecessary functions in `pyccel.parser.utilities` : `read_file`, `header_statement`, `accelerator_statement`, `get_module_name`, `view_tree`.
 -   \[INTERNALS\] Remove unused functions `Errors.unset_target`, and `Errors.reset_target`.
+
+## \[1.12.0\] - 2024-05-08
+
+### Added
+
+-   #1830 : Add a `pyccel.lambdify.lambdify` function to accelerate SymPy expressions.
```

Comment:
Is this not accessible directly as `pyccel.lambdify`?

Reply:
Yes, and it is not accessible as `pyccel.lambdify.lambdify`. This should have been caught in the PR. I will fix it here

Reply:
[d752b18](https://github.com/pyccel/pyccel/pull/1859/commits/d752b181ca240273a3d423acf19cb9d98690bd12)